src/openapi/paths/funderCollaborativeMember.json

```diff
@@ -0,0 +1,150 @@
+{
+	"get": {
+		"operationId": "getFunderCollaborativeMemberByShortCode",
+		"summary": "Gets a specific funder collaborative member.",
+		"tags": ["Funder Collaborative Members"],
+		"security": [
+			{
+				"auth": []
+			}
+		],
+		"parameters": [
+			{
+				"name": "funderCollaborativeShortCode",
+				"description": "The short code of a funder collaborative.",
+				"in": "path",
+				"required": true,
+				"schema": {
+					"type": "string"
+				}
+			},
+			{
+				"name": "memberFunderShortCode",
+				"description": "The short code of funder to be made a member of the collaborative.",
+				"in": "path",
+				"required": true,
+				"schema": {
+					"type": "string"
+				}
+			}
+		],
+		"responses": {
+			"200": {
+				"description": "The funder collaborative member.",
+				"content": {
+					"application/json": {
+						"schema": {
+							"$ref": "../components/schemas/FunderCollaborativeMember.json"
+						}
+					}
+				}
+			},
+			"404": {
+				"$ref": "../components/responses/NotFound.json"
+			}
+		}
+	},
+	"post": {
+		"operationId": "createOrUpdateFunderCollaborativeMember",
+		"summary": "Creates or updates a new funder collaborative member.",
+		"tags": ["Funder Collaborative Members"],
+		"security": [
+			{
+				"auth": []
+			}
+		],
+		"parameters": [
+			{
+				"name": "funderCollaborativeShortCode",
+				"description": "The short code of a collaborative funder.",
+				"in": "path",
+				"required": true,
+				"schema": {
+					"type": "string"
+				}
+			},
+			{
+				"name": "memberFunderShortCode",
+				"description": "The short code of a member non-collaboraitve funder.",
+				"in": "path",
+				"required": true,
+				"schema": {
+					"type": "string"
+				}
+			}
+		],
+		"requestBody": {
+			"required": true,
+			"content": {
+				"application/json": {
+					"schema": {
+						"$ref": "../components/schemas/FunderCollaborativeMember.json"
+					}
+				}
+			}
+		},
+		"responses": {
+			"201": {
+				"description": "The new funder collaborative member that was created.",
+				"content": {
+					"application/json": {
+						"schema": {
+							"$ref": "../components/schemas/FunderCollaborativeMember.json"
+						}
+					}
+				}
+			},
+			"400": {
+				"$ref": "../components/responses/DatabaseError.json"
+			},
+			"401": {
+				"$ref": "../components/responses/Unauthorized.json"
+			}
```

Comment:
I also get a 422 here (currently undocumented) when adding a member that doesn't exist.